docs/app/pages/home/backend-bundles-section/backend-bundles-section.component.html

```diff
@@ -0,0 +1,248 @@
+      </p>
+    </li>
+    <li class="features__item">
+      <h3 class="features__title">Functional</h3>
+      <p class="features__description">Functional Backend Solution with Designed and Implemented Services and Data Flow. Deploy it as ready to use solution for a particular case, or give it to your development team to incrementally add functionality for the bigger system. it gives you a significant boost on start and solid development structure.</p>
+    </li>
+    <li class="features__item">
+      <h3 class="features__title">Efficient</h3>
+      <p class="features__description">Save more than $21,000 using Backend Bundle. According to our research usage of a ready Backend Bundle optimizes 300 hours of development time. Taking the average hourly rate of a full stack developer in the US - $70.</p>
+    </li>
+    <li class="features__item">
+      <h3 class="features__title">Ready to use</h3>
+      <p class="features__description">We prepared this backend pack as development basement which lets your team concentrate on business logic and data models. Now .NET and .NET Core are available as frameworks of choice. More are in progress…</p>
+    </li>
+  </ul>
+
+  <picture class="bundle-scheme">
+    <source type="image/webp" srcset="/assets/img/bundle-scheme@1x.webp 1x, /assets/img/bundle-scheme@2x.webp 2x">
+    <img class="bundle-scheme__image" src="/assets/img/bundle-scheme@1x.png" srcset="/assets/img/bundle-scheme@2x.png 2x" alt="Backend bundles">
+  </picture>
+
+  <div class="package-switcher">
+    <div [(ngModel)]="licenseType" ngbRadioGroup
+         class="package-switcher__btn-group btn-group btn-group-toggle btn-outline-toggle-group btn-group-full-width btn-toggle-radio-group">
+      <label ngbButtonLabel class="package-switcher__btn btn btn-outline-secondary">
+        <input ngbButton type="radio" value="Personal"> Personal
+      </label>
+      <label ngbButtonLabel class="package-switcher__btn btn btn-outline-secondary">
+        <input ngbButton type="radio" value="Commercial"> Commercial
+      </label>
+    </div>
+  </div>
+
+  <ul class="packages">
+    <li class="packages__item">
+      <nb-flip-card [showToggleButton]="false" [flipped]="firstCardFlipped" (click)="firstCardFlipped = !firstCardFlipped">
+        <nb-card-front class="package-card package-card--front">
+          <nb-card>
+            <nb-card-body>
+              <div class="package-card__header">
+                <span class="package-card__demo-link">Coming soon</span>
+              </div>
+              <div class="package-card__image package-card__image--dot-net"></div>
+              <div class="package-card__price-wrapper">
+                <span class="package-card__price package-card__price--old">${{ licenseType === 'Commercial' ? '2000' : '200' }}</span>
```

Comment:
Please, use `isCommercial` method instead of hardcoding strings in the template.